devicetypes/smartthings/zigbee-switch-power.src/zigbee-switch-power.groovy

```diff
@@ -57,8 +57,6 @@ metadata {
 
 		// Enbrighten
 		fingerprint manufacturer: "Jasco Products", model: "43078", deviceJoinName: "Enbrighten Switch" //Enbrighten In-Wall Smart Switch With Energy Monitoring 43078, Raw Description: 01 0104 0100 00 07 0000 0003 0004 0005 0006 0702 0B05 02 000A 0019
```

Comment:
this device's name indicates that it _also_ supports energy monitoring. Are you sure you shouldn't be moving it as well?

Reply:
@greens Yes, I was checking that device yesterday too. 
If it's ok to move a switch to zigbee-metering-plug DTH then I'll do it.